docs/prompts.md

```diff
@@ -17,18 +17,18 @@ approaches are also supported by working with prompts directly.
 ## Defining prompts
 
 Genkit's `generate()` helper function accepts string prompts, and you can
-call models this way for straight-forward use cases.
+call models this way for straightforward use cases.
 
 ```ts
 import { generate } from '@genkit-ai/ai';
 
 generate({
   model: 'googleai/gemini-1.5-flash-latest',
-  prompt: 'You are a helpful AI assistant named Walt.',
+  prompt: 'You are a helpful AI assistant named Walt, say hello',
 });
 ```
 
-In most cases, you will need to include some customer provided inputs in your prompt.
+In some cases you will need to include some customer provided inputs in your prompt.
 You could define a function to render them like this.
```

Comment:
Scratch the function business, and simplify this to be:

> In some cases you will need to include customer provided inputs in your prompt.
You might use Javascript's Template Literals to do this.
>
>  ```ts
> generate({
>   model: 'googleai/gemini-1.5-flash-latest',
>   prompt: `You are a helpful AI assistant named Walt. Say hello to ${name}.`,
> });
> ```

Reply:
Good call, updated